python/ray/serve/schema.py

```diff
@@ -758,6 +759,11 @@ def get_status_dict(self) -> Dict:
         )
 
 
+@PublicAPI(stability="alpha")
+class HTTPProxyDetails(BaseModel):
+    status: HTTPProxyStatus
```

Comment:
can we add node_id, ip_address here?

@alanwguo anything else that there would be useful to show in the UI?

oh... maybe logs path?

Reply:
I've added metadata for http proxy here: https://github.com/ray-project/ray/pull/34944/files#diff-e454e14e8c39af64fcec6997326bd32012fc91afe5d09d6ac1486aed6199afd1R764-R776